frontend/src/components/panels/project_filters_panel/ProjectFiltersPanel.js

```diff
@@ -32,9 +32,9 @@ const ProjectFiltersPanel = () => (
                 <div className="col">
                     <SubHeader Text="Project Type"/>
                     <div className="project-types-container">
-                        <CheckBox Label="Building" ProjectType="building"/>
-                        <CheckBox Label="Vehicle Transportation" ProjectType="vehicle"/>
-                        <CheckBox Label="Infrastructure Transportation" ProjectType="infrastructure"/>
+                        <CheckBoxRow Label="Building" ProjectType="building"/>
```

Comment:
These `Label` values should be dynamically set by all available `project types` values in state.locations.